assets/src/edit-story/app/uploader/useUploader.js

```diff
@@ -47,6 +47,7 @@ function useUploader(refreshLibrary = true) {
       video: allowedVideoMimeTypes,
     },
   } = useConfig();
+  const { allowedFileTypes = {} } = useConfig();
```

Comment:
This can be added to the `useConfig` call above. Also, use `[]` as the default.

Reply:
Done